### PR TITLE
Docs: Update PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #241: Docs: Update pull request template
> [!IMPORTANT]
> If you've worked on a good first issue in any of our [community repositories](https://docs.opensauced.pizza/contributing/opensauced-maintainers-guide/community-maintainers-guide/#opensauced-community-repositories) before, please refrain from taking this issue.
> We may remove you from the assignees and not accept your pull request if you choose to proceed.

## Description

We have updated the "[Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started)" section on our Intro to Open Source course as following:

- Adding contributors to the guestbook with CLI only.
- Encouraging contributors to learn how to create issues.

Therefore, we need to update the PR template to reflect this update.

Additionally, we also want to remind contributors to complete the PR form.

## Suggested Solution

- [ ] Add a comment on the first line with the copy below.

   ```markdown
   <!-- Please fill in all areas in this PR form. Incomplete PRs will be marked invalid and may be closed. -->
   ```

- [ ] Add a new section for related issues right after "What type of PR" section. Use the copy below.

```markdown
## Related Issues

<!-- 
Please use this format link issue numbers: Closes #123
More information: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->
```

- [ ] Update wording in the "Contributing Checklist" section with the below suggestion.

   ```diff
   - ## Contributors checklist (check all applicable)
   + ## Contributors checklist
   ```

   https://github.com/open-sauced/guestbook/blob/20aeca121bd3a794b5c4ec4611f1937b9ac31e5f/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L13

- [ ] Remove the below sections.

   https://github.com/open-sauced/guestbook/blob/20aeca121bd3a794b5c4ec4611f1937b9ac31e5f/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L20-L29

### PR DESCRIPTION
**Pull request by Woytsekj**
## Description
Added comment to the first line to remind contributors to complete the form

Added a new section for related issues right after "What type of PR"

Updated "Contributors checklist (check all applicable)" to "Contributors Checklist"

Removed "How did you add yourself as a contributor?" and "If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?" sections

Updated "https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started" to "https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started"

## What type of PR is this? (check all applicable)

- [ ] 🤝 Add a contributor
- [x] 📝 Documentation Update

## Related Issues
Closes #241

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 PULL_REQUEST_TEMPLATE.md

## Screenshot (Required for PR Review)
image
image
image

## [optional] What GIF best describes this PR or how it makes you feel?
😃

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hi @Woytsekj ,
Thank you for your PR! 🙌

However, please complete your PR form so we can understand and review the changes you made.

That said, please provide a description of the changes that you made,

We're only reviewing this PR once we get all the information needed.
Please let us know once you've completed the form.

Thank you. 🙂

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<!-- Please fill in all areas in this PR form. Incomplete PRs will be marked invalid and may be closed. -->
 ## Description
```
Comment (adiati98) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: Add a new line (space) between line 1 and 2.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -10,22 +11,19 @@ This PR adds <github-username> as a contributor.
...
-
-### How did you add yourself as a contributor?
+<!-- 
+Please use this format link issue numbers: Closes #123
+More information: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
+-->
 
-- [ ] 🤖 With CLI
-- [ ] ⌨️ Manually
+## Contributors checklist
```
Comment (adiati98) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: Let's capitalize this

```suggestion
## Contributors Checklist
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -10,22 +11,19 @@ This PR adds <github-username> as a contributor.
...
 
-#### If you added yourself manually, did you follow the [emoji key and contribution types](https://allcontributors.org/docs/en/emoji-key) to fill in the value?
+### I've read through the [Getting Started](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=getting-started) section.
```
Comment (adiati98) [inline .github/PULL_REQUEST_TEMPLATE.md:23]: We recently updated our course. So, please change the link with the below:

```suggestion
### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -10,22 +11,19 @@ This PR adds <github-username> as a contributor.
...
 
 - [ ] ✅ Yes
-- [ ] ❌ No
+- [ ] ❌ Not yet
 
 ### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?
 - [ ] ✅ Yes
```
Comment (adiati98) [inline .github/PULL_REQUEST_TEMPLATE.md:29]: Add a new line (space) between line 28 and 29.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @Woytsekj,
I've left some reviews here.

Please let me know if you've fixed them.

Thank you.

**Comment by Woytsekj** [comment]
Hi @adiati98,

I have read the reviews and fixed them.
I have also edited my pull request to have a description.

Thank you for reminding me.

**Review by BekahHW — APPROVED** [review]

**Review by adiati98 — APPROVED** [review]

**Pull request merged**